File: src/test/java/ru/AlexeyFedechkin/znatoki/StudyRussianBot/Utils/ChartTest.java
package ru.AlexeyFedechkin.znatoki.StudyRussianBot.Utils;

import org.junit.Test;

import java.io.File;

import static org.junit.Assert.assertTrue;

@SuppressWarnings("HardCodedStringLiteral")
public class ChartTest {

    @Test
    public void genOneLineGraf() {
        double[] xData = new double[]{1, 10};
        double[] yData = new double[]{1, 10};
        File file = new Chart().genOneLineGraf("test", "line", yData, xData);
        assertTrue(file.isFile());
        file.delete();
    }
}

File: src/test/java/ru/alexeyFedechkin/znatoki/studyRussianBot/Utils/ChartTest.kt
package ru.alexeyFedechkin.znatoki.studyRussianBot.Utils

import org.junit.Assert.assertTrue
import org.junit.Test

class ChartTest {
    @Test
    fun genOneLineGraf() {
        val xData = doubleArrayOf(1.0, 10.0)
        val yData = doubleArrayOf(1.0, 10.0)
        val file = Chart.genOneLineGraf("test", "line", yData, xData)
        assertTrue(file.isFile)
        file.delete()
    }
}